### PR TITLE
Add BackgroundBlurDeviceProvider and VideoInputBackgroundBlurControl to meeting demo

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:**
https://github.com/aws/amazon-chime-sdk-js/issues/1690

**Description of changes:**
Add the `BackgroundBlurDeviceProvider` to the main component tree and `VideoInputBackgroundBlurControl` to the `MeetingControls`.

This depends on the 2.11.0 version of the Amazon Chime React SDK Component Library and also the 2.20.0 version of the Amazon Chime SDK for Javascript. Both have not been released yet, so this PR will sit in PR until both are released. 

**Testing**

1. How did you test these changes?

```
1. npm run start
2. join a meeting
3. toggle video on and off
4. enable bg blur
5. toggle video on and off
6. switch between videos
```

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
-  "name": "amazon-chime-sdk",
-  "lockfileVersion": 2,
-  "requires": true,
-  "packages": {}
-}
```
Comment (michhyun1) [inline package-lock.json:6]: why is there a package-lock on the root level of this repo? We can delete it like this right?

File: apps/meeting/src/containers/MeetingControls/index.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {
   ControlBar,
   AudioInputVFControl,
   AudioInputControl,
-  VideoInputControl,
+  VideoInputBackgroundBlurControl,
```
Comment (devalevenkatesh) [inline apps/meeting/src/containers/MeetingControls/index.tsx:9]: We do not want to remove the existing `VideoInputControl`, but provide a check to use background blur feature like we do for VF now.

File: apps/meeting/src/containers/MeetingProviderWrapper/index.tsx [hunk 3]
```diff
@@ -26,24 +27,26 @@ const MeetingProviderWrapper: React.FC = () => {
 
   const getMeetingProviderWrapper = () => {
     return (
-      <MeetingProvider {...meetingConfigValue}>
```
Comment (devalevenkatesh) [inline apps/meeting/src/containers/MeetingProviderWrapper/index.tsx:29]: We do not want to change this right, but, add another method like `getMeetingProviderWrapperWithVF` say `getMeetingProviderWrapperWithBgBlur`. We want to test the demo such that we should be able to test demo with only bg blur, only VF, with VF and bgblur and without anything with simple controls without these features.

**Review by devalevenkatesh — CHANGES_REQUESTED** [review]

File: apps/meeting/src/containers/MeetingControls/index.tsx (hunk 2 above)
Comment (michhyun1) [reply apps/meeting/src/containers/MeetingControls/index.tsx:9]: done

**Pull request merged**